console/src/schematic/ontology.tsx

```diff
@@ -98,9 +97,8 @@ const TreeContextMenu: Ontology.TreeContextMenu = ({
 
   const handleRename = (): void => Tree.startRenaming(resources[0].key);
 
-  const clusterKey = Cluster.useSelectActiveKey();
   const handleCopyUrl = (): void => {
-    const url = `synnax://cluster/${clusterKey}/schematic/${resources[0].id.key}`;
+    const url = `synnax://cluster/${client.key}/schematic/${resources[0].id.key}`;
```

Comment:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.